test/mpi/part/testlist

```diff
@@ -0,0 +1,31 @@
+start_pready 2 arg=-spartitions=1 arg=-rpartitions=1 arg=-tot_count=64 arg=-range=1 arg=-iteration=5
+start_pready 2 arg=-spartitions=8 arg=-rpartitions=8 arg=-tot_count=64 arg=-range=2 arg=-iteration=5
+start_pready 2 arg=-spartitions=64 arg=-rpartitions=64 arg=-tot_count=64 arg=-range=4 arg=-iteration=5
+start_pready 2 arg=-spartitions=512 arg=-rpartitions=512 arg=-tot_count=4194304 arg=-range=64 arg=-iteration=5
+start_pready 2 arg=-spartitions=1 arg=-rpartitions=16 arg=-tot_count=64 arg=-range=1 arg=-iteration=5
+start_pready 2 arg=-spartitions=16 arg=-rpartitions=32 arg=-tot_count=64 arg=-range=2 arg=-iteration=5
+start_pready 2 arg=-spartitions=8192 arg=-rpartitions=512 arg=-tot_count=4194304 arg=-range=64 arg=-iteration=5
+
```

Comment:
`count ` might be confusing as it is the total count but not per-partition-count. OK to shorten the other args.

Reply:
Found a bit hard to replace `iteration`->`iter`, because internally use `iter` as the current iteration id. I think it is fine to keep `iteration` as the arg name.